tests/test_awkward.py

```diff
@@ -58,3 +59,22 @@ def test_awkward_registration():
     arr = pylhe.to_awkward(pylhe.read_lhe_with_attributes(TEST_FILE_WITH_WEIGHTS))
     assert len(arr.particles.vector.mass) == len(arr.particles)
     assert len(arr.particles.vector.mass) == len(arr.weights)
+
+
+def test_to_awkward_vector():
+    """
+    Test numeric equality of momentas represented by vector.
```

Comment:
```suggestion
    Test numeric equality of momenta represented by vectors.
```

Reply:
Thanks. Since this was unlikely to be the only spelling error I added the codespell precommit hook and fixed the errors it showed.